terraform_fortios_provider/TEMP_Cert_1.pem

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
can we generate this pem instead?

Reply:
Don't know exactly why this is here, but originally we were required by the config to add a temp cert which then gets overwritten by the root in FortiDemo.

Might be able to generate now with the change?

Reply:
I got this to work by using FOS CSF default SAML configuration. Now the FortiDemo idp certificate will be obtained by downstream AWS FGT using the csfd daemon